objectRecognitionSource/src/main/java/edu/ml/tensorflow/util/ImageUtil.java

```diff
@@ -115,6 +168,21 @@ public void saveImage(final BufferedImage image, final String target) {
             }
         }
     }
+    
+    /**
+     * A function used to resize the original captured image, if requested.
+     * @param width     The resized width of the image to be saved.
+     * @param height    The resized height of the image to be saved.
+     * @param image     The original image to be resized.
+     * @return          The resized image.
+     */
+    public BufferedImage resizeImage(int width, int height, BufferedImage image) {
```

Comment:
maybe we could have a variant that includes the longEdge that does all that work.

Reply:
Yep that's exactly what I'm doing now.